py-polars/polars/dataframe/frame.py

```diff
@@ -3436,14 +3436,15 @@ def write_database(
         elif engine == "sqlalchemy":
             if parse_version(pd.__version__) < parse_version("1.5"):
                 raise ModuleNotFoundError(
-                    f"writing with engine 'sqlalchemy' requires pandas 1.5.x or higher, found pandas {pd.__version__!r}"
+                    f"writing with engine `sqlalchemy` requires pandas 1.5.x or higher, found pandas {pd.__version__!r}"
```

Comment:
sqlalchemy isn't a code snippet - it's a name of a package. It shouldn't get backticks.

Reply:
Understood. Whilst editing I saw that packages are sometimes within single quotes, sometimes with CamelCase notation. Any standardization you would like there as well?